app/src/main/java/com/github/fribourgsdp/radio/User.kt

```diff
@@ -0,0 +1,52 @@
+package com.github.fribourgsdp.radio
+
+import android.graphics.Color
+import java.lang.Exception
+import java.lang.IllegalArgumentException
+import kotlin.random.Random
+
+class User (nameInput: String) {
+    var name: String = nameInput
+        set(value) {
+            nameSanitizer(value)
+            field = value
+        }
+    init {
+        nameSanitizer(name)
+    }
+    private val playlists = mutableSetOf<Playlist>()
+    val color : Int = Color.rgb(
+        Random.nextInt(100, 200),
+        Random.nextInt(100, 200),
+        Random.nextInt(100, 200))
+    private var linkedSpotify: Boolean = false
```

Comment:
Isn't it a bit early to add this variable regarding spotify since it is not yet integrated ? Remember "code for what you need, not what you may need in the future"

Reply:
Yes you are correct, this will definitely change. This was just a way to test for a dynamic UI